Tella/Scenes/HomeView/ViewModel/FileListViewModel.swift

```diff
@@ -389,3 +389,62 @@ extension FileListViewModel {
         return FileListViewModel(appModel: MainAppModel.stub(), filterType: .all, rootFile: VaultFileDB.stub())
     }
 }
+
+extension FileListViewModel {
+    
+    func getFilesInFolder(parentId: String) ->  Int{
+        let files =  appModel.vaultFilesManager?.getVaultFiles(parentId: parentId, filter: self.filterType, sort: self.sortBy) ?? []
+        return files.count
+    }
+
+    var deleteConfirmationTitle: String {
+        let folderCount = selectedFiles.filter { $0.type == .directory }.count
+        let fileCount = selectedFiles.count - folderCount
+
+        switch (fileCount, folderCount) {
+        case (0, 1):
+            return LocalizableVault.deleteFolderSheetTitle.localized
+        case (0, _):
+            return LocalizableVault.deleteFoldersSheetTitle.localized
+        case (1, 0):
+            return LocalizableVault.deleteFileSheetTitle.localized
+        default:
+            return LocalizableVault.deleteFilesSheetTitle.localized
+        }
+    }
+
+    var deleteConfirmationMessage: String {
+        let selectedFolders = selectedFiles.filter { $0.type == .directory }
+        let folderCount = selectedFiles.filter { $0.type == .directory }.count
+        let fileCount = selectedFiles.count - folderCount
+        
+        let totalFilesInsideFolders = selectedFolders.reduce(0) { count, folder in
+            count + getFilesInFolder(parentId: folder.id ?? "")
```

Comment:
I don't think that this implementation will give you all the files under the selected folders, it will give you only the first files under the folder
I suggest to use `FileWalker` to get all the files, it's used in 
`func deleteVaultFile(vaultFiles : [VaultFileDB]) -> Result<Bool, Error>?`

Reply:
ohh right, i forgot about nested folders. I will implement this rn!